sites/all/themes/wille/template.php

```diff
@@ -45,6 +45,20 @@ function wille_preprocess_wille_site_template(&$variables, $hook) {
 }
 
 /**
+ * Implements THEME_preprocess_TEMPLATE();
+ *
+ * We preprocess and override the ctools content type here in the theme, because
+ * there are theming specific files and styles that shoudl be included.
+ */
+function wille_preprocess_breol_news_page(&$variables, $hook) {
+  $variables['organic_svg'] = file_get_contents(dirname(__FILE__) . "/svg/organic.svg");
+  if ($fid = variable_get('breol_news_page_image', NULL)) {
```

Comment:
Shouldn't be needed. $vars['image_file'] should already be the file object, added by breol_news_page_news_page().